README.md

```diff
@@ -1,4 +1,75 @@
-# Easy Connect - Easily add all supported connectivity methods to your mbed OS project
+# Easy Connect - Easily add all supported connectivity methods to your Mbed OS project
+
+## Deprecation note
+
+Please note that starting with [Mbed OS 5.10](https://github.com/ARMmbed/mbed-os/tree/mbed-os-5.10) onwards the network driver can be instantiated using `get_default_instance()`. 
+
+**This repository will not be maintained going onwards, as the work will be done using the official Mbed OS repository.**
+
```

Comment:
This repository will not be maintained as the work continues in the official Mbed OS repository (maybe a link to it).